themes/SuiteR/tpls/Home.tpl

```diff
@@ -70,11 +70,11 @@
 
 
 <!-- CSS Files -->
-<link type="text/css" href="{sugar_getjspath file='custom/include/SugarCharts/js/Jit/Examples/css/base.css'}" rel="stylesheet" />
-<link type="text/css" href="{sugar_getjspath file='custom/include/SugarCharts/js/Jit/css/Examples/BarChart.css'}" rel="stylesheet" />
-<!--[if IE]><script language="javascript" type="text/javascript" src="{sugar_getjspath file='custom/include/SugarCharts/js/Jit/Extras/excanvas.js'}"></script><![endif]-->
+<link type="text/css" href="{sugar_getjspath file='include/SugarCharts/js/Jit/Examples/css/base.css'}" rel="stylesheet" />
```

Comment:
still incorrect should be include/SugarCharts/Jit/css/base.css?, but is probably legacy and should be removed as with below